internal/envoy/bootstrap.go

```diff
@@ -92,6 +98,18 @@ func (c *BootstrapConfig) GetAdminAccessLogPath() string {
 	return stringOrDefault(c.AdminAccessLogPath, "/dev/null")
 }
 
+// GetXdsVersion returns the version if specified or an error if invalid value.
+// Defaults to "v2".
+func (c *BootstrapConfig) GetXdsVersion() (config.ServerVersion, error) {
+	switch c.XDSServerVersion {
```

Comment:
should be able to use pkg/config/ServerVersion.Validate() here to avoid duplicating the validation

Reply:
ahh right, good call. =)